superflore/generators/bitbake/run.py

```diff
@@ -31,38 +32,29 @@
 from superflore.utils import info
 from superflore.utils import load_pr
 from superflore.utils import ok
+from superflore.utils import ros2_distros
 from superflore.utils import save_pr
 from superflore.utils import url_to_repo_org
 from superflore.utils import warn
 
-# Modify if a new distro is added
-active_distros = ['indigo', 'kinetic', 'lunar']
-
 
 def main():
-    preserve_existing = True
     overlay = None
+    preserve_existing = True
```

Comment:
to reduce the delta, does the location of this variable need to change?

Reply:
Right, otherwise it's a small one line delta.